conda_build/skeletons/cran.py

```diff
@@ -92,6 +92,10 @@
   rpaths:
     - lib/R/lib/
     - lib/
+
+  # This is needed to suppress a warning about R.dll not being found
+  missing_dso_whitelist:
+    - '*/R.dll'            # [win]
```

Comment:
Same goes for `Rblas.dll` and `Rlapack.dll`, which are not infrequent.

```suggestion
    - '*/R.dll'            # [win]
    - '*/Rblas.dll'        # [win]
    - '*/Rlapack.dll'      # [win]
```